pkg/operator/ceph/object/bucket/s3-handlers.go

```diff
@@ -314,18 +316,22 @@ func (ps *PolicyStatement) WithSID(sid string) *PolicyStatement {
 }
 
 const awsPrinciple = "AWS"
+const arnPrefix = "arn:aws:"
 
 // ForPrincipals adds users to the PolicyStatement
 func (ps *PolicyStatement) ForPrincipals(users ...string) *PolicyStatement {
-	ps.Principal[awsPrinciple] = users
+	principals := ps.Principal[awsPrinciple]
+	for _, u := range users {
+		principals = append(principals, fmt.Sprintf("%siam:::user/%s", arnPrefix, u))
```

Comment:
should this fmt string "%siam:::user/%s" be a const?

Reply:
an alternative style is to fix up the `users` slice with the formatting and then assign `ps.Principal[awsPrinciple] = users` as done originally. This changes fewer lines but is really not much different from your fix.